google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/AddConversionBasedUserList.java

```diff
@@ -0,0 +1,147 @@
+import com.google.ads.googleads.v2.enums.UserListMembershipStatusEnum.UserListMembershipStatus;
+import com.google.ads.googleads.v2.errors.GoogleAdsError;
+import com.google.ads.googleads.v2.errors.GoogleAdsException;
+import com.google.ads.googleads.v2.resources.UserList;
+import com.google.ads.googleads.v2.services.MutateUserListsResponse;
+import com.google.ads.googleads.v2.services.UserListOperation;
+import com.google.ads.googleads.v2.services.UserListServiceClient;
+import com.google.ads.googleads.v2.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.Int64Value;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/** Creates a basic user list consisting of people who triggered one or more conversion actions. */
+public class AddConversionBasedUserList {
+  private static class AddConversionBasedUserListParams extends CodeSampleParams {
+
+    @Parameter(names = ArgumentNames.CUSTOMER_ID, required = true)
+    private Long customerId;
+
+    @Parameter(names = ArgumentNames.CONVERSION_ACTION_ID, required = true)
+    private List<Long> conversionActionIds;
+  }
+
+  public static void main(String[] args) {
+    AddConversionBasedUserListParams params = new AddConversionBasedUserListParams();
+    if (!params.parseArguments(args)) {
+
+      // Either pass the required parameters for this example on the command line, or insert them
+      // into the code here. See the parameter class definition above for descriptions.
+      params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+      params.conversionActionIds =
+          Arrays.asList(
+              Long.parseLong("INSERT_CONVERSION_ACTION_ID_HERE"),
+              Long.parseLong("INSERT_CONVERSION_ACTION_ID_HERE"));
+    }
+
+    GoogleAdsClient googleAdsClient;
+    try {
+      googleAdsClient = GoogleAdsClient.newBuilder().fromPropertiesFile().build();
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      return;
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      return;
+    }
+
+    try {
+      new AddConversionBasedUserList()
+          .runExample(googleAdsClient, params.customerId, params.conversionActionIds);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
```

Comment:
This is a good idea, we should get this in other libraries too. Pls add a line to the front log.